README.md

```diff
@@ -1,53 +1,56 @@
-MultiLineString, MultiPolygon, GeometryCollection. Next to a geometry, simple features
+MultiLineString, MultiPolygon, and GeometryCollection. Next to a geometry, simple features
 can also have non-spatial attributes that describe the feature.
 
 Geospatial data often comes in tabular format, with one or more columns with
 feature geometries and additional columns with feature attributes. The Arrow columnar
-memory model is therefore perfectly suited to store such data, both vector features and
+memory model is well-suited to store both vector features and
 their attribute data. The GeoArrow specification defines how the vector features
 (geometries) can be stored in Arrow (and Arrow-compatible) data structures.
 
 This repository contains the specifications for:
 
-- The memory layout for storing geometries in an Arrow field ([format.md](./format.md))
-- The Arrow extension type definitions ([extension-types.md](./extension-types.md))
+- The memory layout for storing geometries in an Arrow array ([format.md](./format.md))
+- The Arrow extension type definitions that ensure type-level metadata (e.g., CRS) is
+  propagated when used in Arrow implementations ([extension-types.md](./extension-types.md))
 
 Defining a standard and efficient way to store geospatial data in the Arrow memory
-layout helps interoperability between different tools and ensures geospatial tools can
+layout enables interoperability between different tools and ensures geospatial tools can
 leverage the growing Apache Arrow ecosystem:
 
 - Efficient, columnar file formats. Leveraging the performant and compact storage of
   Apache Parquet as a vector data format in geospatial tools using
-  [GeoParquet](https://github.com/opengeospatial/geoparquet/).
+  [GeoParquet](https://github.com/opengeospatial/geoparquet/)
 - Accelerated between-process geospatial data exchange using Apache Arrow IPC message
   format and Apache Arrow Flight
 - Zero-copy in-process geospatial data transport using the Apache Arrow C Data Interface
   (e.g., GDAL)
 - Shared libraries for geospatial data type representation and computation for query
-  engines that support columnar data formats (e.g., Velox, DuckDB, Acero)
-
-The goal of this repository is to discuss and move towards specifications
-on metadata and memory layout for storing geospatial data in the Arrow memory
-layout or related formats.
+  engines that support columnar data formats (e.g., Velox, DuckDB, and Acero)
 
 ### Relationship with GeoParquet
 
 The GeoParquet specification originally started in this repo, but was moved out into its
 own repo (https://github.com/opengeospatial/geoparquet), leaving this repo to focus on
-the Arrow-specific specifications (Arrow layout and extension type metadata).
+the Arrow-specific specifications (Arrow layout and extension type metadata). Whereas
+GeoParquet is a file-level metadata specification, GeoArrow is an array-level metadata
```

Comment:
```suggestion
GeoParquet is a file-level metadata specification, GeoArrow is a field-level metadata
```

Is that strictly speaking more correct? (but I am certainly fine with having it as "array", or "column")